Kodilla-testing/src/main/java/com/kodilla/testing/forum/statistics/StatisticsApp.java

```diff
@@ -0,0 +1,64 @@
+package com.kodilla.testing.forum.statistics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StatisticsApp implements Statistics {
```

Comment:
Nie implementuje statistics

Reply:
Tak, poprawiłem to od razu ale jakoś nie rozumiem, dlaczego nie ma tej zmiany.